drop-types/src/ffi.rs

```diff
@@ -0,0 +1,113 @@
+use std::time::Duration;
+
+use serde::Deserialize;
+
+#[derive(Deserialize, Clone, Debug)]
+pub struct Config {
```

Comment:
I don't like the idea of `ffi` module here and perhaps leave all ffi related stuff in `norddrop` crate. But if you insist, let's move it out of this module and rename it to `ConfigJson` or something like that, because the purpose of this structure is to define the JSON representation of the actual config.